### PR TITLE
Link README with MI Indicators

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -18,6 +18,11 @@ matplotlib = "*"
 pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
+hypothesis-auto = "*"
+hypothesis = "*"
+pytest-mypy = "*"
+mypy = "*"
+pre-commit = "==2.0.1"
```
Comment (xtuchyna) [inline Pipfile:25]: why is pre-commit pinned down to this specific version?

File: Pipfile [hunk 2]
```diff
@@ -18,6 +18,11 @@ matplotlib = "*"
 pytest = "*"
 pytest-timeout = "*"
 pytest-cov = "*"
+hypothesis-auto = "*"
+hypothesis = "*"
```
Comment (xtuchyna) [inline Pipfile:22]: What is the use case of hypothesis module?

**Review by xtuchyna — COMMENTED** [review]
Some comments regarding new modules, otherwise LGTM :+1:

File: Pipfile (hunk 2 above)
Comment (pacospace) [reply Pipfile:22]: https://hypothesis.readthedocs.io/en/latest/

File: Pipfile (hunk 1 above)
Comment (pacospace) [reply Pipfile:25]: It should not be pinned, nice catch!!

**Review by xtuchyna — APPROVED** [review]
LGTM

**Pull request merged**